extensions/2.0/Khronos/KHR_audio/README.md

```diff
@@ -0,0 +1,314 @@
+# KHR_audio
+
+## Contributors
+
+* Robert Long, Element Inc.
+* Anthony Burchell, Individual Contributor
+* K. S. Ernest (iFire) Lee, Individual Contributor
+* Michael Nisbet, Individual Contributor
+* humbletim, Individual Contributor
+* Norbert Nopper, UX3D [@UX3DGpuSoftware](https://twitter.com/UX3DGpuSoftware)
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension allows for the addition of spatialized and non-spatialized audio to glTF scenes.
+
+Audio emitter objects may be added to 3D nodes for positional audio or to the scene for environmental or ambient audio such as background music.
+
+### Example:
+
+```json
+{
+  "extensions": {
+    "KHR_audio": {
+      "emitters": [
+        {
+          "name": "Positional Emitter",
+          "type": "positional",
+          "gain": 0.8,
+          "sources": [0, 1],
+          "positional": {
+            "coneInnerAngle": 6.283185307179586,
+            "coneOuterAngle": 6.283185307179586,
+            "coneOuterGain": 0.0,
+            "distanceModel": "inverse",
+            "maxDistance": 10.0,
+            "refDistance": 1.0,
+            "rolloffFactor": 0.8
+          }
+        }
+      ],
+      "sources": [
+        {
+          "name": "Clip 1",
+          "gain": 0.6,
+          "autoPlay": true,
+          "loop": true,
```

Comment:
These 3 properties are in the example and in the schema, but they are not defined in the README.